readthedocs/analytics/signals.py

```diff
@@ -19,8 +19,9 @@ def increase_page_view_count(sender, **kwargs):
     version = context['version']
     # footer_response sends an empty path for the index
     path = context['path'] or '/'
+    page_slug = context['page_slug']
 
-    if project.has_feature(Feature.STORE_PAGEVIEWS):
+    if page_slug != "404" and project.has_feature(Feature.STORE_PAGEVIEWS):
```

Comment:
Can be the path different if is it a 404? I think we can just use the path to check this, `page` is a sphinx only concept, and I think mkdocs doesn't set that anymore.

Reply:
Hmmm, I didn't test it on mkdocs but you're right that this won't work. I'm not exactly sure what we should test for in that case. Both the `page_slug` and the `path` will be blank on mkdocs in that case.

Reply:
Yea, we need more than this I imagine, I think we need the actual status code passed back to the backend. I'm not really sure how to actually get the original status code in JS tho :/

Reply:
Just for confirmation, this is a request from a mkdocs project, page is empty.

```
https://mkdocs-clone.readthedocs.io/_/api/v2/footer_html/?callback=jQuery110207075518632941309_1592257832592&project=mkdocs-clone&version=latest&page=&theme=mkdocs&format=jsonp&docroot=docs&source_suffix=.md&_=1592257832593
```

Reply:
After looking, I don't believe there is a JS API to get the current page's status code.